custom_components/unifi_insights/api/network/endpoints/lags.py

```
@@ -0,0 +1,200 @@
+        response = await self._client._get(path)
+
+        if isinstance(response, dict):
+            data = response.get("data", response)
+            if isinstance(data, dict):
+                return McLagDomain.model_validate(data)
+            if isinstance(data, list) and len(data) > 0:
+                return McLagDomain.model_validate(data[0])
+        raise ValueError(f"MC-LAG domain {mc_lag_domain_id} not found")
+
+
+async def _list_paginated(
```

Pylint (4.0.6)

[refactor] Too many arguments (6/5)

(R0913)